testcases/kernel/syscalls/send/send01.c

```diff
@@ -103,6 +103,7 @@ static struct test_case_t tdat[] = {
 	,
 #ifndef UCLINUX
 	/* Skip since uClinux does not implement memory protection */
+#if 0 // Enable when sgx-lkl github issue 772 is fixed. Actually maybe never due to PROT_NONE not being detectable
```

Comment:
I think this comment is misleading. I think it would be better to say that it is off because we have the same memory protection as uClinux.

Reply:
I don't know if we do. It is mostly that the memory is protected but by the host OS not sgx-lkl and sgx-lkl in lkl_access_ok cannot tell if a page is disallowed for access. I guess that uclinux doesn't have memory protection at all. So while the effect is the same I think the reasons are a bit different.